interface/ceed-basis.c

```diff
@@ -621,6 +625,75 @@ int CeedBasisCreateH1(Ceed ceed, CeedElemTopology topo, CeedInt num_comp,
   return CEED_ERROR_SUCCESS;
 }
 
+/**
+  @brief Create a non tensor-product basis for H(div) discretizations
+  @param ceed        A Ceed object where the CeedBasis will be created
+  @param topo        Topology of element, e.g. hypercube, simplex, ect
+  @param num_comp    Number of componenet, we have 1 vector componenet in H(div)
```

Comment:
```suggestion
  @param num_comp    Number of components (usually 1 for vectors in H(div) bases)
```
VSCode has a spell checker for code that helps to find typos